frontend/src/DefaultMakeStylesTheme.js

```diff
@@ -0,0 +1,14 @@
+import { makeStyles } from "@material-ui/core";
+
+export default function useStyles() {
```

Comment:
this gives a code smell because the function name doesn't match the file name, but i'm happy with how descriptive the file name is.
should i just change the function name and make sure we use `import * as useStyles`?